### ISSUE TITLE
1.63 is now not opening saves

### ISSUE THREAD

**Issue by Coulbauth** [wmtorode/MW5-SaveEditor#27]
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/12459763/MW5SELog.txt)

**Comment by wmtorode** [comment]
As noted in the release notes, I was unable to fix these reports as I can't replicate it with any save I have. please attached the problematic save here so that I can look further into it

**Comment by Coulbauth** [comment]
I went and tried to add some srm 2's to my save, but it was in an exiting the game save and it wigged out. I went back into the game created a new save and it worked fine but the auto still fails.
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/12490643/MW5SELog.txt)

**Comment by wmtorode** [comment]
@Coulbauth for the second time, I can't replicate this locally, I need the failing save, otherwise there is nothing I can do

**Comment by Coulbauth** [comment]
hope this helps


[933E84D84E7D4ED4AD71888870EE411B.zip](https://github.com/wmtorode/MW5-SaveEditor/files/12501789/933E84D84E7D4ED4AD71888870EE411B.zip)

**Comment by AenjoDave** [comment]
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/12784451/MW5SELog.txt)
[041E0BA64473647877348D8DAA26F808.zip](https://github.com/wmtorode/MW5-SaveEditor/files/12784480/041E0BA64473647877348D8DAA26F808.zip)

Another example.

Thanks for the work you put into this.  Made my MW5 experience much better.

**Comment by JacobSKessler** [comment]
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/12797322/MW5SELog.txt)
Thanks for keeping this alive

**Comment by PainGod85** [comment]
And another one.
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/13062044/MW5SELog.txt)

Saves from before DLC5 load without issue for me.

**Comment by PainGod85** [comment]
[0B71B51C464E2D957A68EE889470C910.zip](https://github.com/wmtorode/MW5-SaveEditor/files/13062046/0B71B51C464E2D957A68EE889470C910.zip)

Forgot to attach the problematic save in question.

**Comment by fetusyeetus** [comment]
I think I've figured out the "File "libmw5save\pilotroster.py"" issue, as it's happening to me too. If I start a new game I can open and edit the save as normal. The issue seems to be the mod Pilot Promotion from Nexus. I promoted one of my pilots and was no longer able to open the save. Log and problematic save attached.
[MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/13255817/MW5SELog.txt)
[85AFC83A41DCB4BE091903A3E7B179C7.zip](https://github.com/wmtorode/MW5-SaveEditor/files/13255818/85AFC83A41DCB4BE091903A3E7B179C7.zip)

**Comment by Toony24** [comment]
yeah pilot promotion seemed to be the cause of the break for me too. however it might be time for us to give up and look elsewhere, if there even is another. this project is likely dead and abandoned again, there hasn't been an update or even a reply since august it seems.

**Comment by wmtorode** [comment]
Just really busy these days, don't have much time to look at this but want to get back to it at some point

**Comment by eric7237cire** [comment]
> I think I've figured out the "File "libmw5save\pilotroster.py"" issue, as it's happening to me too. If I start a new game I can open and edit the save as normal. The issue seems to be the mod Pilot Promotion from Nexus. I promoted one of my pilots and was no longer able to open the save. Log and problematic save attached. [MW5SELog.txt](https://github.com/wmtorode/MW5-SaveEditor/files/13255817/MW5SELog.txt) [85AFC83A41DCB4BE091903A3E7B179C7.zip](https://github.com/wmtorode/MW5-SaveEditor/files/13255818/85AFC83A41DCB4BE091903A3E7B179C7.zip)

Thank you !

This gives a workaround, if you dismiss all of your pilots you promoted with the pilot promotion mod, then the save editor is able to open the sav again.

**Comment by wmtorode** [comment]
these isses should hopefully be fixed in the new 1.6.4 release
